var/spack/repos/builtin/packages/meme/package.py

```diff
@@ -35,9 +35,13 @@ class Meme(AutotoolsPackage):
 
     version('4.11.4', '371f513f82fa0888205748e333003897')
 
+    variant('mpi', default=True, description='Enable MPI support')
+
     depends_on('zlib', type=('link'))
     depends_on('libxml2', type=('link'))
     depends_on('libxslt', type=('link'))
     depends_on('libgcrypt', type=('link'))
     depends_on('perl', type=('build', 'run'))
     depends_on('python@2.7:', type=('build', 'run'))
+    # openmpi support
+    depends_on('openmpi@2:', type=('link'))
```

Comment:
Both of those are the same thing. Here are some examples to illustrate why we support both:
```
meme-mpi   # a package named meme-mpi
meme -mpi  # a package named meme, without mpi support
meme~mpi   # a package named meme, without mpi support
meme ~mpi  # same as `meme /home/mpi`
```
You only need to check for one, not both. @tgamblin can you confirm?

> ```
> spack install meme -mpi
> ```

Are you sure that's the exact command you ran? Something doesn't add up here.